tutorials/use-cloud-pubsub-cloud-storage-app-engine/index.md

```diff
@@ -64,7 +64,7 @@ A general overview of how the application works is shown in the diagrams below.
 
 ![Loading Notifications](https://storage.googleapis.com/gcp-community/tutorials/use-cloud-pubsub-cloud-storage-app-engine/loading-home-page.png)
 
-1.  The user navigates to `http://[YOUR_PROJECT_ID].appspot.com/`.
+1.  The user navigates to `[YOUR PROJECT ID].appspot.com`.
```

Comment:
Please revert back to `[YOUR_PROJECT_ID]`, this is the standard placeholder variable we use throughout the documentation on cloud.google.com.

Otherwise LGTM.